sky/shell/gpu/direct/rasterizer_direct.cc

```diff
@@ -53,6 +53,7 @@ void RasterizerDirect::OnAcceleratedWidgetAvailable(gfx::AcceleratedWidget widge
   surface_ =
       gfx::GLSurface::CreateViewGLSurface(widget, gfx::SurfaceConfiguration());
   CHECK(surface_) << "GLSurface required.";
+  EnsureGLContext();
```

Comment:
I might add a comment to explain why we do this eagerly.  Someone could find it via git blame of course.